automation/run-tests.sh

```diff
@@ -259,11 +259,16 @@ while true; do
         shift
         EXPORT_DIR="$1"
         ;;
+    --test-vdsm)
+        shift
+        echo "Testing vdsm"
+        exit
+        ;;
     --help)
         set +x
         echo -n "$0 [--copr=...] [--customize=...] [--debug-shell] [--el8] "
         echo -n "[--help] [--pytest-args=...] "
-        echo "[--test-type=<TEST_TYPE>]"
+        echo "[--test-type=<TEST_TYPE>] [--test-vdsm=<PR_NUMBER>]"
```

Comment:
It would be great if this could use the local Nmstate git checkout, maybe by adding a volume to the VDSM container. Then it would also work on locally instead of only for published PRs.

Reply:
I was thinking about that, let's see what I will manage to do with vdsm side